pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -591,6 +591,9 @@ type NodeGroup struct {
 	// +optional
 	Taints map[string]string `json:"taints,omitempty"`
 
+	// +optional
+	LoadBalancerNames []string `json:"loadBalancerNames,omitempty"`
```

Comment:
Should we specify that this is only for CLBs by calling it instead `clasicLBNames` for example? Or will people understand that?

Reply:
`LoadBalancerNames` is the same as the property name of CloudFormation API ( https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html ), but as you say `clasicLBNames` looks clearer.

Reply:
Thank you very much for this PR @af12066! 🙇🏻‍♂✨  
We should avoid the typo (`clasicLBNames` -> `classicLBNames`), but sounds good otherwise! 👍
A shame CloudFormation API's isn't more explicit as it would have been nice to be consistent with it, indeed.

Reply:
While CloudFormation uses `LoadBalancerNames`, and we have used the same property name as CloudFormation for `TargetGroupARNs`, I think we should be more explicit in eksctl by calling it `classicLBNames` (or `classicLoadBalancerNames`) but mention that it maps to the `LoadBalancerNames` property in CloudFormation.

Reply:
+1 for `classicLoadBalancerNames` -- asEasierToRead 👀

Reply:
Thanks for reviewing. I'll fix it.